### PR TITLE
[doc] Move Usage section in Install to own page

### PR DESCRIPTION
**Pull request by JJRcop**
Also includes a few miscellaneous strawberry flavoured formatting changes sprinkled on top to taste.

### CODE REVIEW

File: doc/usage.rst [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+.. _client_config_options_file:
+
+Configuration Files
+~~~~~~~~~~~~~~~~~~~
+
+By default, Looking Glass will load config files from
+the following locations:
+
+-  /etc/looking-glass-client.ini
+-  ~/.looking-glass-client.ini
```
Comment (JJRcop) [inline doc/usage.rst:97]: Were these deprecated? I heard something about that from someone on discord

Comment (gnif) [reply doc/usage.rst:97]: Yes, they still work but we want people to move across to using the XDG_CONFIG path.

Comment (JJRcop) [reply doc/usage.rst:97]: Alright, should I add a notice here?

Comment (gnif) [reply doc/usage.rst:97]: I know I said otherwise earlier, but on 2nd thoughts we will keep it, let people decide

**Pull request merged**